src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+
+    if (this.health > 0) {
+      Animal.alive.push(this);
```

Comment:
The logic for adding animals to the `Animal.alive` array should be reviewed. Consider implementing a method within the `Animal` class to manage the `alive` array more robustly, ensuring only living animals are tracked and removed when their health reaches zero.